runtime/doc/starting.txt

```diff
@@ -106,8 +106,8 @@ argument.
 		This can be used to find out where time is spent while loading
 		your |config|, plugins and opening the first file.
 		When {fname} already exists new messages are appended.
-		(Only available when compiled with the |+startuptime|
-		feature).
+		{only available when compiled with the |+startuptime|
```

Comment:
Should I just remove this instead? It seems that other mentions of compile-time features are removed from docs.

Reply:
Sure.